packages/create-llama/helpers/run-app.ts

```diff
@@ -1,8 +1,22 @@
-import { ChildProcess, spawn } from "child_process";
+import { ChildProcess, SpawnOptions, spawn } from "child_process";
 import { log } from "console";
 import path from "path";
 import { TemplateFramework } from "./types";
 
+const createProcess = (
+  command: string,
+  args: string[],
+  options: SpawnOptions,
+) => {
+  return spawn(command, args, {
+    ...options,
+    shell: /^win/.test(process.platform),
```

Comment:
@leehuwuj @thucpn We didn't use `shell` in `callpackageManager` (and it works on windows):

https://github.com/run-llama/LlamaIndexTS/blob/656bb56e91d0c6f51f61a1b97f572cbbdd197609/packages/create-llama/helpers/install.ts#L31-L41

another difference is that it's using
`process.chdir(props.root);` instead of the `cwd` argument - maybe that has an error on windows? or the path is not resolved correctly?